go.md

```diff
@@ -324,7 +324,9 @@ Inhaltliche Änderungen wurden vorgenommen auf der:
 - Sommer-ZaPF 2019 in Bonn,
 - Winter-ZaPF 2019 in Freiburg,
 - der Sommer-ZaPF 2023 in Berlin,
-- und auf Winter-ZaPF 2023 in Düsseldorf.
+- der Winter-ZaPF 2023 in Düsseldorf,
+- der Sommer-ZaPF 2024 in Kiel
```

Comment:
In der anderen PR vielleicht untergegangen, aber wenn ich durch die Commithistory schaue, sehe ich nicht, was wir da geändert hätten?